### ISSUE TITLE
Traceback calling certain three.js methods: "JsException: TypeError: jsobj.constructor is undefined"

### ISSUE THREAD

**Issue by tildebyte** [pyodide/pyodide#2528]
## 🐛 Bug

I'm assuming that this is a bug and not a missing feature...
e.g.
```python
from js import THREE
GREY = THREE.Color.new('rgb(70, 71, 76)')
```

is perfectly happy.

OTOH,

```python
from js import THREE
foo = THREE.MathUtils.randFloat(0.75, 1.5)
```
 is NOT at all:

### Firefox
```python
# pyodide.asm.js:14:237960
Python exception:
Traceback (most recent call last):
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 2, in <module>
JsException: TypeError: jsobj.constructor is undefined
```
```js
// error_handling.gen.ts:309:4
Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 2, in <module>
JsException: TypeError: jsobj.constructor is undefined
    PythonError error_handling.gen.ts:309
    new_error pyodide.asm.js:14
    _pythonexc2js pyodide.asm.js:14
    callPyObjectKwargs pyproxy.gen.ts:374
    callPyObject pyproxy.gen.ts:384
    apply pyproxy.gen.ts:1145
    apply pyproxy.gen.ts:1022
    runPython api.ts:71
    main pyscript-three-bug-minimal.html:11
    async* pyscript-three-bug-minimal.html:13
```

### Chromium Ungoogled (weirdly different from the FF TB)

```python
# pyodide.asm.js:14
Python exception:
Traceback (most recent call last):
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 2, in <module>
JsException: TypeError: Cannot read properties of undefined (reading 'name')


Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 2, in <module>
JsException: TypeError: Cannot read properties of undefined (reading 'name')

    at new_error (pyodide.asm.js:14:238191)
    at pyodide.asm.wasm:0xedbcb
    at pyodide.asm.wasm:0xedccc
    at Object.Module._pythonexc2js (pyodide.asm.js:14:932707)
    at Object.Module.callPyObjectKwargs (pyproxy.gen.ts:374:12)
    at Object.Module.callPyObject (pyproxy.gen.ts:384:17)
    at PyProxyClass.apply (pyproxy.gen.ts:1145:19)
    at Object.apply (pyproxy.gen.ts:1022:18)
    at Object.runPython (api.ts:71:25)
    at main (pyscript-three-bug-minimal.html:11:29)
```
### To Reproduce

```html
<!DOCTYPE html>
<html>
  <head>
      <script src="https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.min.js"></script>
  </head>
  <body>
    <script type="text/javascript">
      async function main(){
        let pyodide = await loadPyodide();
        console.log(pyodide.runPython("from js import THREE\nfoo = THREE.MathUtils.randFloat.new(0.75, 1.5)"));
      }
      main();
    </script>
  </body>
</html>
```

### Expected behavior

`foo` is assigned the value returned by the JS function call

### Environment

- Pyodide Version
		cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js
- Browser version
		Firefox 100.0 64-bit; Chromium Ungoogled 101.0.4951.54 64-bit
- Any other relevant information
		three.js version cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.min.js

### Additional context
Messing around with pyscript; took a while to figure out that it was probably in the pyodide layer

**Comment by hoodmane** [comment]
I think this is fixed by #2520.

**Comment by hoodmane** [comment]
Thanks for the report!

**Comment by tildebyte** [comment]
Nice! Thanks for the quick reply.

**Comment by hoodmane** [comment]
Closed by #2520 (I think).
